dd-sdk-android/src/main/json/log/log-schema.json

```diff
@@ -0,0 +1,174 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "log-schema.json",
+  "title": "LogEvent",
+  "type": "object",
+  "description": "Structure holding information about a Log",
+  "properties": {
+    "status": {
+      "type": "string",
+      "description": "The severity of this log",
+      "enum": [
+        "critical",
+        "error",
+        "warn",
+        "info",
+        "debug",
+        "trace",
+        "emergency"
+      ],
+      "readOnly": true
+    },
+    "service": {
+      "type": "string",
+      "description": "The service name",
+      "readOnly": false
+    },
+    "message": {
+      "type": "string",
+      "description": "The log message",
+      "readOnly": false
+    },
+    "logger": {
+      "type": "object",
+      "description": "Information about the logger that produced this log.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the logger",
+          "readOnly": false
+        },
+        "thread_name": {
+          "type": "string",
+          "description": "The thread name on which the log event was created",
+          "readOnly": true
+        },
+        "version": {
+          "type": "string",
+          "description": "The SDK version name",
+          "readOnly": true
+        }
+      },
+      "required": [
+        "name",
+        "thread_name",
+        "version"
+      ],
+      "readOnly": true
+    },
+    "usr": {
+      "type": "object",
+      "description": "User properties",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Identifier of the user",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the user",
+          "readOnly": true
+        },
+        "email": {
+          "type": "string",
+          "description": "Email of the user",
+          "readOnly": true
+        }
+      },
+      "additionalProperties": {
```

Comment:
just out of curiosity: why sometimes we have `additionalProperties` with the `usr` and sometimes don't? for example, long task event doesn't have it.

Reply:
Normally they all should have this `additionalAttributes` as this is part of the public API. I guess it must be an issue in the `LongTaskEvent` 🤔

Reply:
It's not an issue with long task but with RUM. There's some discussion on that topic on how we store custom RUM data